api-reference/10 UI Components/dxForm/5 Item Types/SimpleItem/aiOptions/aiOptions.md

```diff
@@ -4,7 +4,96 @@ type: Object
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies options for AI service when you implement Smart Paste.
```

Comment:
Technically, our customers do not "implement" Smart Paste. Part of that functionality is implemented by us (DevExpress). The part that parses some text into individual fields is implemented by the AI Service. 

```suggestion
Specifies options for an AI service that parses Smart Paste data.
```

Our customers can "activate" Smart Paste.